### ISSUE TITLE
Prioritize bootstrapping by vote weight?

### ISSUE THREAD

**Issue by qwahzi** [nanocurrency/nano-node#2134]
Moving [this comment](https://github.com/nanocurrency/nano-node/issues/1731#issuecomment-501247765) from #1731 to its own issue.

Has there been any discussion of prioritizing bootstrapping by vote weight for bandwidth-constrained circumstances? If the network is under heavy load, it seems to make sense to have PRs focus on bootstrapping other PRs vs wasting bandwidth on bootstrapping $5/mo Digital Ocean nodes.

@schenkty

**Comment by Falsen** [comment]
Just a thought I have: Isn't it quite rare for a PR having much voting power falling behind? Is it really worth the effort

**Comment by schenkty** [comment]
No, that isn’t the point. The point is PR’s having to help worthless nodes while under load and having it’s bandwidth limited without any benefit to the PR. If a PR is behind, they probably shouldn’t be a PR.

**Comment by Falsen** [comment]
Ah I see, yes absolutely! Bootstraping any other noode should really be low priority under heavy load

**Comment by zhyatt** [comment]
We will be reviewing this as part of two-phase voting and network overlay, as the desired behavior for bootstrapping will shift with these changes.
